### PR TITLE
Master

### PR DESCRIPTION
**Pull request by RasulovaB**
Please review measure.js file.

### CODE REVIEW

File: testBR.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        </div>
+    </div>
+    
+    <section class="about" id="about">
+    
+        
+    <h2  id="date"> </h2>
+    </section>
+    
+    <script src="moment.js"></script>
```
Comment (matthewrhoden1) [inline testBR.html]: Remove this line and delete the moment.js file from the repository.

File: measure.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+//var x = myFunction(1, 0.5);
+//document.getElementById("test").innerHTML = x;
+//
+//function myFunction(a, b) {
+//  return a + b;
+//}  
+    
+function calcG(){
+    //take the value and save the variable to x
+    x=document.getElementById("kilo").value;
```
Comment (matthewrhoden1) [inline measure.js:21]: Please answer the following questions BEFORE running this code:

```
"use strict";
function test(){
    y = 123;
}
test();
```

1. What would be the tokens of this line `y = 123;`
2. Referencing chapter 1, explain what you expect to happen if you were to try to execute the above code block.
3. Open dev tools `F12` paste this into the `Console` tab and observe the result. 
4. Come back and make any code changes necessary to this calcG function.

**Review by matthewrhoden1 — CHANGES_REQUESTED** [review]
I think the best way to do this is through questions and answers. I'll ask you questions to nudge you in the right direction. If it worked for Socrates, it should work for us 👍 

Just remember, put your school first, I won't get all judgy if it takes you a month to respond to this.

File: measure.js (hunk 2 above)
Comment (RasulovaB) [reply measure.js:21]: Your questions helped me to understand what I've read in the book. Thank you! 🙏 
Btw, I didn’t have a chance to learn how to use Git. If I am doing something wrong with branches, pulling request, and comments, please forgive me.

**Pull request merged**